### PR TITLE
Documentation for OpenStack provider

### PR DESCRIPTION
**Pull request by zreigz**

### CODE REVIEW

**Comment by cheld** [comment]
Nice docs

**Comment by zreigz** [comment]
I will work on travis to fix the problem

**Comment by zreigz** [comment]
Ready to merge

File: docs/getting-started-guides/openstack.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
```
Comment (ohtake) [inline docs/getting-started-guides/openstack.md]: No corresponding `strong` open tag. The following code is missing?

```
<!-- TAG RELEASE_LINK, added by the munger automatically -->
<strong>
```

File: docs/getting-started-guides/openstack.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    heat >= 0.9.0
+
+    swift >= 2.7.0
+
+    glance >= 1.2.0
+
+    nova >= 3.2.0
+
+    ```
+    sudo apt-get remove python-hetaclient
```
Comment (ohtake) [inline docs/getting-started-guides/openstack.md]: Typo? It must be `python-heatclient`. See ling 60 too.

File: docs/getting-started-guides/openstack.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+### Interacting with your Kubernetes cluster with Openstack
+
+You can manage the nodes in your cluster with the Openstack WEB UI like Horizon or using heat or nova clients.
+
+To get all necessary information about cluster execute commands in main kubernetes directory:
+
+```
+. cluster/openstack/openrc-default.sh
+. cluster/openstack/openrc-default.sh
```
Comment (ohtake) [inline docs/getting-started-guides/openstack.md]: Duplicated.

**Comment by zreigz** [comment]
Thanks for comments. I will fix it in new pull request

**Pull request merged**